daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -11,7 +11,8 @@ class RazerViperMini(__RazerDevice):
     """
     Class for the Razer Viper Mini
     """
-    EVENT_FILE_REGEX = re.compile(r'.*Razer_Razer_Viper_Mini-if0(1|2)-event-kbd')
+    EVENT_FILE_REGEX = re.compile(
+        r'.*Razer_Razer_Viper_Mini-if0(1|2)-event-kbd')
```

Comment:
Please revert all these changes that have nothing to do with this device. Keep the formatting for your additions aligned to how existing code looks.

Reply:
Ah, yeah, that's my autoformatting at work. Will fix that.